css/css-ruby/reference/rt-dynamically-appended-001-ref.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Reference File</title>
+
+  <style>
+  body
+    {
+      font-size: 60px;
+    }
+
+  ruby
+    {
+      background-color: lightblue;
+    }
+  </style>
+
+  <ruby>bas<rt>T</rt>&nbsp;<rt>X</rt></ruby>
```

Comment:
I think the test fails in Chromium because of this extra `&nbsp;` in the reference file, not because of the dynamic insertion. If you remove it here, Chromium should just pass, albeit in a wrong way.

Reply:
Xidorn,
If I remove the `&nbsp;`, then I will create a false positive for Chromium 94 (and lower; and presumably Safari 14.x) or contribute to make unreliable test result for Chromium... which is already the case for dozens of /css/css-ruby tests.

As far as I understand the ruby spec, the current reference file and the current test should be rendered exactly the same.

Reply:
I can remove dynamic insertion from the test, rename files and have the code of the test and the code of reference file be exactly the same, except for a single `&nbsp;` which would be in the reference file, just like it is now.

Reply:
If your target is that it should fail Chromium because it handles unpaired ruby annotation in a wrong way, so do it with a test checking specifically that. If your target is to test that dynamic inserting a ruby annotation should end up in the same rendering as when it's statically present, then test that. Don't mix them together to artificially fail Chromium because of the the former on a test which is seemingly checking the latter.

Reply:
Fair enough. I will refocus the target of the test accordingly: mishandling unpaired ruby annotation.